src/dmd/globals.d

```diff
@@ -27,12 +27,15 @@ template xversion(string s)
 
 enum IN_GCC     = xversion!`IN_GCC`;
 
-enum TARGET_LINUX   = xversion!`linux`;
-enum TARGET_OSX     = xversion!`OSX`;
-enum TARGET_FREEBSD = xversion!`FreeBSD`;
-enum TARGET_OPENBSD = xversion!`OpenBSD`;
-enum TARGET_SOLARIS = xversion!`Solaris`;
-enum TARGET_WINDOS  = xversion!`Windows`;
+enum TARGET
+{
+    linux   = xversion!`linux`,
```

Comment:
Capital `L' for Linux?

Reply:
I kept the same smelling as `version(...)` uses to avoid confusions.

Reply:
All other `version`s and `Target`s use proper case.  It appears that `version(linux)` was a mistake.  I vote to not proliferate that mistake.

Reply:
Ok. Fair enough ;-)